test/recipes/70-test_sslsigalgs.t

```diff
@@ -132,35 +132,56 @@ SKIP: {
 }
 
 SKIP: {
-    skip "EC or TLSv1.2 disabled", 8 if disabled("tls1_2") || disabled("ec");
+    skip "EC or TLSv1.2 disabled", 10 if disabled("tls1_2") || disabled("ec");
 
     $proxy->filter(\&sigalgs_filter);
 
-    #Test 10: Sending no sig algs extension in TLSv1.2 should succeed
+    #Test 10: Sending no sig algs extension in TLSv1.2 should succeed at
+    #         security level 1
     $proxy->clear();
     $testtype = NO_SIG_ALGS_EXT;
-    $proxy->clientflags("-no_tls1_3");
-    $proxy->ciphers("ECDHE-RSA-AES128-SHA");
+    $proxy->clientflags("-no_tls1_3 -cipher DEFAULT\@SECLEVEL=1");
+    $proxy->ciphers("ECDHE-RSA-AES128-SHA\@SECLEVEL=1");
```

Comment:
Isn't this the default SECLEVEL anyway - so why does it need to be explicitly stated?

Reply:
Do you want the tests to pass if someone builds with a different seclevel as the default?

Reply:
#10786 will need to change all the values to be 1 lower. It's also explicit at which level we expect to succeed and which to fail, we might at some point want to change the default security level.